### PR TITLE
3057927 kount merchant

### PR DESCRIPTION
**Pull request by safalliajoseph**
* Add settings in store for kount merchant ID
* Add kount merchant ID to iframe
d.o - https://www.drupal.org/project/commerce_bluesnap/issues/3057927

### CODE REVIEW

**Comment by safalliajoseph** [comment]
@krystalcode I have done the modifications as we discussed yesterday. Could you please review

**Comment by krystalcode** [comment]
@safalliajoseph I have made a few changes, please review my commits and the commits' comments describing the changes. Let me know if it looks good so that we can merge.

File: src/FraudPrevention/FraudSession.php [hunk 1]
```diff
@@ -57,42 +56,28 @@ public function remove() {
...
+    $params = $this->iframeParams($kount_merchant_id);
     $iframe = '
       <iframe
         width="1"
         height="1"
         frameborder="0"
         scrolling="no"
-        src="{{ url }}/servlet/logo.htm?s={{ fraud_session_id }}">
-        <img width="1" height="1" src="{{ url }}"/servlet/logo.gif?s={{ fraud_session_id }}>
+        src="{{ url }}/servlet/logo.htm?' . $params . '">
```
Comment (safalliajoseph) [inline src/FraudPrevention/FraudSession.php]: @krystalcode Instead of string concatenation could we not pass $params as context?

**Review by safalliajoseph — COMMENTED** [review]
@krystalcode I tested locally and everything looks good, I modified the `iframeParams` method a bit. Please review

**Comment by safalliajoseph** [comment]
@krystalcode I have modified the code to store the kount merchant ID in store config, Please review

**Comment by krystalcode** [comment]
Merged on Drupal.org

**Pull request closed without merging**